src/main/java/com/faforever/client/fx/WindowController.java

```diff
@@ -1,7 +1,7 @@
 package com.faforever.client.fx;
 
 import com.faforever.client.theme.UiService;
-import javafx.collections.ObservableList;
+import com.sun.javafx.util.Utils;
```

Comment:
(Almost) all `com.sun` are JVM internal and do not exist on other JVM implementations. Also, such package names may change with any update without prior notice, breaking compatibility ([like it happened to ControlsFX]("com.sun.javafx.Utils" changes to "com.sun.javafx.util.Utils" in 1.8.0_60ea))

This, this import should be removed. Keep the old code or, if easy, copy the code from `Utils.getVisualBounds` into our `JavaFxUtil`